app/src/main/res/layout/item_creating_left_conversation.xml

```diff
@@ -22,16 +24,17 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginBottom="@dimen/dp_8"
-        android:layout_marginEnd="@dimen/dp_56"
-        android:layout_marginStart="@dimen/dp_8">
+        android:layout_marginEnd="@dimen/dp_56">
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 16) you should *also* add `android:layout_marginRight="@dimen/dp_56"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.